app/Main.hs

```diff
@@ -32,6 +33,11 @@ contracts = daoContractRegistry
     , dciConfig = GameDAO.config
     , dciExtraParser = pure def
     }
+  , DaoContractInfo
+    { dciName = "RegistryDAO"
+    , dciConfig = RegistryDAO.config @ByteString @ByteString
+    , dciExtraParser = pure def
+    }
```

Comment:
I think we should also update CI to upload Registry DAO documentation. I propose to add a comment somewhere around this list to remind about that.